src/GO/Job.php

```diff
@@ -498,7 +500,7 @@ private function finalise()
     /**
      * Email the output of the job, if any.
      *
-     * @return void
+     * @return bool
```

Comment:
I'd like to keep it as void, I guess there is no difference between returning nothing or returning always `true`, unless you have an argue that's better returning true.
PS: remove also the `return true;` at the end of this fn thanks